models/provisioning-jobs.ts

```diff
@@ -72,8 +79,8 @@ export const provisionRequestSchema = {
       required: ["name", "uri", "network"],
```

Comment:
If `uri` and `network` are deprecated should they be required still?

Reply:
Oh uhh no -- sorry I missed updating that. I changed the TS interface and didn't see this field.